sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -82,13 +81,20 @@ export class StreamingReceiver extends MessageReceiver {
     if (options.autoComplete == null) options.autoComplete = true;
     const sReceiver = new StreamingReceiver(context, options);
 
+    sReceiver._init = async function() {
+      console.log("Faking it!");
+      throw new MessagingError("Hello there, I'm an error");
+    };
     const config: RetryConfig<void> = {
       operation: () => {
         return sReceiver._init();
       },
       connectionId: context.namespace.connectionId,
       operationType: RetryOperationType.receiveMessage,
-      retryOptions: options.retryOptions
+      retryOptions: {
+        maxRetries: 2,
+        retryDelayInMs: 0
```

Comment:
if `retryDelayInMs: 0` is removed, the `_init` operation doesn't retry which is not the expected behavior.